.nuget/NuGet.Config

```diff
@@ -14,4 +14,7 @@
   <packageSources>
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
   </packageSources>
+  <bindingRedirects>
+    <add key="skip" value="True" />
+  </bindingRedirects> 
```

Comment:
I'm just curious why you chose to skip adding the binding redirects for the sln.